src/util/util.c

```diff
@@ -40,7 +40,13 @@ int mph_to_kmph(int mph)
 {
 	// 1 mph = 1.60934 kmph
 	// RCT2 approximates as 1.609375
-	return (mph * 1648) / 1024;
+	return (mph * 1648) >> 10;
```

Comment:
I think this optimisation should be left up to the compiler.The intent seems more clear in the original code.